tools/c7n_left/c7n_left/core.py

```diff
@@ -336,8 +336,18 @@ def as_dict(self):
         }
 
 
+class LeftValueFilter(ValueFilter):
+    def get_resource_value(self, k, i):
+        if k.startswith('tag:') and 'tags' in i:
+            tk = k.split(':', 1)[1]
+            r = (i.get('tags', {}) or {}).get(tk)
+            return r
+        return super().get_resource_value(k, i)
+
+
 class IACResourceManager(ResourceManager):
     filter_registry = FilterRegistry("iac.filters")
+    filter_registry.register('value', LeftValueFilter)
```

Comment:
It looks like this isn't covering the shorthand form of a value filter. So this still uses the core `ValueFilter`:

```yaml
"filters": [{"tag:Env": "absent"}],
```

But this uses `LeftValueFilter`:

```yaml
"filters": [{"type": "value", "key": "tag:Env", "value": "absent"}]
```

I'm not sure offhand what else we need to change/add to have it use `LeftValueFilter` across the board, but wanted to at least raise a flag for the moment.

Reply:
fixed